src/scripts/main.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const tBody = document.querySelector('tbody');
+const min = 2;
+const max = 10;
```

Comment:
Constants are uppercase
```suggestion
const MIN = 2;
const MAX = 10;
```